webapp/tsconfig.json

```diff
@@ -2,17 +2,18 @@
   "compileOnSave": false,
   "compilerOptions": {
     "baseUrl": "./",
+    "downlevelIteration": true,
     "outDir": "./dist/out-tsc",
     "sourceMap": true,
     "declaration": false,
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
`ESNext` is quite bleeding edge, you're sure you want that?

Reply:
has been done by `ng update`, reverted back to `commonjs`.